src/app/services/messages.service.ts

```diff
@@ -154,8 +172,9 @@ export class MessagesService {
     public getAllMessages(isPublished?: boolean, observe?: 'events', reportProgress?: boolean): Observable<HttpEvent<Array<Message>>>;
     public getAllMessages(isPublished?: boolean, observe: any = 'body', reportProgress: boolean = false ): Observable<any> {
 
+
         let queryParameters = new HttpParams({encoder: new CustomHttpUrlEncodingCodec()});
-        if (isPublished !== undefined) {
+        if (isPublished !== undefined && isPublished !== null) {
```

Comment:
Could we just combine the checks for null and undefined as per the first answer here: https://stackoverflow.com/questions/5515310/is-there-a-standard-function-to-check-for-null-undefined-or-blank-variables-in?

Reply:
I think this is autogenerated

Reply:
Ah, right, okay no problem